### DISCUSSION TITLE
JoCodes Breakdown

### DISCUSSION THREAD

**Discussion by Roboslime** [DynamicTreesTeam/DynamicTrees#733] [Q&A] [answered]
So, the JoCodes section of the Wiki is either out of date for 1.16.5, or it doesn't tell the full story. Anybody willing to go into a full breakdown of exactly what they mean? I get the encoding/decoding between Hex and Base64, but what exactly each command does doesn't seem to correspond to what I see in game...and the vast inconsistency between them is also annoying.

A staff made by commands will make the same tree as a command to make that tree, but the commands to make a tree seem to make any already existing tree regress in growth, and a staff used on a generated tree, even by the staff itself, doesn't make the same tree.

**Comment by supermassimo** [comment, marked as answer]
Might be random to answer a post this old but for anyone else who wonders, the wiki has been updated with more info about jocodes https://github.com/DynamicTreesTeam/DynamicTrees/wiki/JoCodes
